### DISCUSSION TITLE
Lines

### DISCUSSION THREAD

**Discussion by HanaO00** [davatorium/rofi#1458] [Theming]
I configured rofi but the "Lines" option does not work if it is in the "Configuration" block, is it normal? It works well by being placed  on the "*" bloc
![ok](https://user-images.githubusercontent.com/17810985/132967539-ab943ed5-0096-40bb-905b-1ad1002f2ff9.png)

**Comment by DaveDavenport** [comment]
Please see the release notes and the many other topics on it in this discussion place.
f.e. https://github.com/davatorium/rofi/discussions/1407
